docs/user_guide/HOWTO/Upgrade_SIMP.rst

```diff
@@ -99,3 +99,23 @@ of your new Puppet server into the existing environment.
 
    You should *always* start migration with a small number of
    **least critical** clients!
+
+Significant Updates
+-------------------
+
+Upgrading From SIMP-6.0.0-0 to A Later Version
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Puppetserver auth.conf
+""""""""""""""""""""""
+
+Legacy auth.conf has been deprecated.  The puppetserver's auth.conf is now
+managed by Puppet.
+
+1. Back up legacy puppet auth.conf ``/etc/puppetlabs/puppet/auth.conf``, before
```

Comment:
You don't actually have to do this. Our module will back it up and move it out of the way for you.